apps/namadillo/src/atoms/validators/functions.ts

```diff
@@ -86,13 +86,13 @@ export const toUnbondingValidators = (
       apr
     );
     const withdrawTime = Number(indexerUnbond.withdrawTime);
-    const currentTime = Number(indexerUnbond.currentTime);
-    const secondsLeft = withdrawTime - currentTime;
 
-    // TODO: later return from the backend
-    const canWithdraw = secondsLeft <= 0;
+    const canWithdraw = indexerUnbond.canWithdraw;
     const timeLeft =
-      canWithdraw ? "" : singleUnitDurationFromInterval(timeNow, withdrawTime);
+      // If can't withdraw but estimation is incorrect display n/a
+      !canWithdraw && withdrawTime < timeNow ? "N/A"
```

Comment:
What do you think about this?

![Screenshot 2024-08-07 at 11 11 11](https://github.com/user-attachments/assets/54665bf4-7a31-483c-bca4-0a3cfb3bfdcd)

```ts
const timeLeft =
  canWithdraw ? ""
  : withdrawTime < timeNow ? `Epoch ${indexerUnbond.withdrawEpoch}`
  : singleUnitDurationFromInterval(timeNow, withdrawTime);
```

Reply:
Looks good to me. I thought about it, but I wasn't sure how useful this is because we do not display current epoch anywhere in the interface :D

Reply:
Yeah, I was searching for a place with the current epoch as well 😅 
But at least it give some information, even though it'll be hard to understand